### DISCUSSION TITLE
New Scroll-linked CSS animations look stunning!

### DISCUSSION THREAD

**Discussion by Frakko** [EddieHubCommunity/support#1069] [Show and tell]
I run in to this article, which goes over a new experimental CSS feature: scroll-linked animations.

This offers a way to create animations that are linked to a scroll offset of a scroll container with pure CSS and without the need of any JavaScript - in my opinion they look amazing!

As this is an experimental feature, only Chromium browsers support it as of now (from what I know) and  "Experimental Web Platform Features" needs to be enabled from the browser flag settings - that can be found at this ULR for Chrome -> chrome://flags/ - to make it work.

Here is the article that goes over the topic in depth: https://www.bram.us/2021/02/23/the-future-of-css-scroll-linked-animations-part-1/

Update: I also found this video that goes over it: https://www.youtube.com/watch?v=EkEyA4RtfNE

**Comment by eddiejaoude** [comment]
Looks good 👍  thanks for sharing

**Comment by Vyvy-vi** [comment]
These look cool
